ipykernel/comm/comm.py

```diff
@@ -21,15 +21,22 @@ class Comm(LoggingConfigurable):
     shell = Instance('IPython.core.interactiveshell.InteractiveShellABC',
                      allow_none=True)
     kernel = Instance('ipykernel.kernelbase.Kernel')
-    def _kernel_default(self):
+
+    @default('kernel')
+    def _default_kernel(self):
```

Comment:
This feels redundant - since anonymous functions aren't really a thing in Python, we're likely going to keep naming them like this anyway. So the decorator is just duplicating information. If that's how traitlets works now, though, I guess that's what we'll do. :neutral_face: